patches/api/0318-Add-Git-information-to-version-command-on-startup.patch

```diff
@@ -0,0 +1,164 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Professor Bloodstone <git@bloodstone.dev>
+Date: Sun, 20 Jun 2021 01:48:31 +0200
+Subject: [PATCH] Add Git information to version command/on startup
```

Comment:
Why new patch?
Initially considered putting it in `Version Command 2.0` patch, but since I not only modify it, I decided to put it in a standalone patch.

Reply:
Just fixup and add yourself as coauthor.

Reply:
I meant - since I modify more than just the command, I decided to use a new patch. If you think it belongs in that patch, I don't mind doing that.